java/README.md

```diff
@@ -12,10 +12,17 @@ A jar file will be created at `./build/libs/protobuf-specs-<version>.jar`
 
 ## Releasing
 
+### Generate Release artifacts
 1. On creation of a tag in the style `release/java/v1.2.3`, new artifacts will be built and
 uploaded to a github release `release/java/v1.2.3`
-2. TODO: Explain how a releaser can then complete the release process on their machine by signing
-with pgp and uploading to maven central.
+1. Once a release is created, check it and remove the draft label on the github release page.
+1. On a machine with your pgp key, `gpg`, `bash` and `cosign`, go to `protobuf-specs/java/scripts`
+1. Run `./sign_and_bundle_release.sh v1.2.3` to generate a release bundle for `release/java/v1.2.3`
+
+### Publish on Maven Central
+1. Log into https://s01.oss.sonatype.org with credentials that have permissions to upload to `dev.sigstore`
```

Comment:
Can we describe who has such credentials and how they're managed?

(I need to know who to bother to get a release out, esp. if you're on vacation or something)

Reply:
done